datastore/additional_data/sources/geo_lookup.py

```diff
@@ -0,0 +1,135 @@
+import csv
+import io
+
+import requests
+
+from additional_data.models import GeoLookup
+
+
+class GeoLookupSource(object):
+    """Imports geographical lookups from https://github.com/drkane/geo-lookups/
+    These allow for looking up from lower-level geography like Ward to a standard local authority, region, etc"""
+
+    # Source URLS
+    SOURCE_URLS = {
+        "lsoa": {
+            "url_lookup": "https://raw.githubusercontent.com/drkane/geo-lookups/master/lsoa_la.csv",
+            "url_latlong": "https://raw.githubusercontent.com/drkane/geo-lookups/master/lsoa_latlong.csv",
+            "field_areacode": "lsoa11cd",
+            "field_areaname": "lsoa11nm",
+            "field_transforms": {"lad20cd": "ladcd", "lad20nm": "ladnm",},
+        },
+        "msoa": {
+            "url_lookup": "https://raw.githubusercontent.com/drkane/geo-lookups/master/msoa_la.csv",
+            "url_latlong": "https://raw.githubusercontent.com/drkane/geo-lookups/master/msoa_latlong.csv",
+            "field_areacode": "msoa11cd",
+            "field_areaname": "msoa11hclnm",
+            "field_transforms": {"lad20cd": "ladcd", "lad20nm": "ladnm",},
+        },
+        "ward": {
+            "url_lookup": "https://raw.githubusercontent.com/drkane/geo-lookups/master/ward_all_codes.csv",
+            "url_latlong": "https://raw.githubusercontent.com/drkane/geo-lookups/master/ward_latlong.csv",
+            "field_areacode": "wdcd",
+            "field_areaname": "wdnm",
+        },
+        "la": {
+            "url_lookup": "https://raw.githubusercontent.com/drkane/geo-lookups/master/la_all_codes.csv",
+            "field_areacode": "ladcd",
+            "field_areaname": "ladnm",
+            "field_transforms": {"lad20cd": "ladcd", "lad20nm": "ladnm",},
+        },
+    }
+
+    def __init__(self):
+        pass
+
+    def get_lookup_data(self, areatype, a, areas):
+        def clean_fields(row, a):
+            for k, v in row.items():
+                k = k.lower()
```

Comment:
Would it be possible to give a more meaningful name to `k`, `v`, `a`? Also in the other functions.